libkbfs/interfaces.go

```diff
@@ -107,6 +107,12 @@ type KBFSOps interface {
 	// isn't favorited.
 	DeleteFavorite(ctx context.Context, fav Favorite) error
 
+	// GetOrInitializeNewMDMaster gets the existing MD or initialize a new MD for
+	// master branch
+	GetOrInitializeNewMDMaster(ctx context.Context, mdops MDOps, h *TlfHandle) (
```

Comment:
Previously we worked to remove all references from metadata from the `KBFSOps` interface -- we don't really want outside libraries to know about that concept.  So ideally neither `ImmutableRootMetadata` or the term "MD" would appear in this interface.

I can think of two possible alternatives:
* Define some new interface containing just this method, make it available on `Config`, and set it to the same `*KBFSOpsStandard` instance whenever a `Config` is made.
* When `KeybaseDaemonRPC` is constructed, pass it a function that calls that method on `KBFSOpsStandard`.  (And maybe don't capitalize the method name.)

What do you think?  Other options?  I slightly prefer the second option.